collection/rancher/v2.x/profile-collector/continuous_profiling.sh

```diff
@@ -188,14 +189,31 @@ while getopts "a:p:d:s:t:h" opt; do
   esac
 done
 
-timestamp() {
-  date "+%Y-%m-%d %H:%M:%S"
+loglevel() {
+  kubectl -n $NAMESPACE get pods -l app=$APP --no-headers -o custom-columns=name:.metadata.name | while read rancherpod; do
+    techo Setting $rancherpod $1 logging
+    kubectl -n $NAMESPACE exec $rancherpod -c rancher -- loglevel --set $1
+  done
```

Comment:
Suggestion: rename this to something evidently Rancher specific (eg. `set_rancher_log_level`) and use `app=rancher` instead of `app=$APP` to make what you are doing very evident